core/src/test/java/com/linecorp/armeria/client/EndpointTest.java

```diff
@@ -454,9 +455,10 @@ void equals() {
         final Endpoint a1 = Endpoint.of("a");
         final Endpoint a2 = Endpoint.of("a");
 
-        assertThat(a1).isNotEqualTo(new Object());
-        assertThat(a1).isEqualTo(a1);
-        assertThat(a1).isEqualTo(a2);
+        new EqualsTester()
```

Comment:
Addressed the new assertion of https://errorprone.info/bugpattern/TruthSelfEquals